roles/magnum/templates/etc/magnum/magnum.conf

```diff
@@ -0,0 +1,224 @@
+#topic = magnum-conductor
+
+# RPC timeout for the conductor liveness check that is used for bay
+# locking. (integer value)
+#conductor_life_check_timeout = 4
+
+
+[database]
+
+connection=mysql://magnum:{{ secrets.db_password }}@{{ endpoints.db }}/magnum?charset=utf8
+
+[docker]
+
+# Docker remote api version. Override it according to specific docker
+# api version in your environment. (string value)
+#docker_remote_api_version = 1.17
+
+# Default timeout in seconds for docker client operations. (integer
+# value)
+#default_timeout = 10
+
+# If set, ignore any SSL validation issues (boolean value)
+#api_insecure = false
+
+# Location of CA certificates file for securing docker api requests
+# (tlscacert). (string value)
+#ca_file = <None>
+
+# Location of TLS certificate file for securing docker api requests
+# (tlscert). (string value)
+#cert_file = <None>
+
+# Location of TLS private key file for securing docker api requests
+# (tlskey). (string value)
+#key_file = <None>
+
+
+[glance_client]
+
+# Region in Identity service catalog to use for communication with the
+# OpenStack service. (string value)
+#region_name = <None>
+
+# Type of endpoint in Identity service catalog to use for
+# communication with the OpenStack service. (string value)
+#endpoint_type = publicURL
+
+
+[heat_client]
+
+# Region in Identity service catalog to use for communication with the
+# OpenStack service. (string value)
+#region_name = <None>
+
+# Type of endpoint in Identity service catalog to use for
+# communication with the OpenStack service. (string value)
+#endpoint_type = publicURL
+
+# Optional CA cert file to use in SSL connections. (string value)
+ca_file = {{ magnum.cafile }}
+
+[keystone_authtoken]
+
+identity_uri = {{ endpoints.identity_uri }}
+auth_uri = {{ endpoints.auth_uri }}
+admin_tenant_name = service
+admin_user = magnum
+admin_password = {{ secrets.service_password }}
+cafile = {{ magnum.cafile }}
+
+
+[kubernetes]
+
+# Default protocol of k8s master endpoint (http or https). (string
+# value)
+#k8s_protocol = http
+
+# Default port of the k8s master endpoint. (integer value)
+#k8s_port = 8080
+
+
+[magnum_client]
+
+# Region in Identity service catalog to use for communication with the
+# OpenStack service. (string value)
+#region_name = <None>
+
+# Type of endpoint in Identity service catalog to use for
+# communication with the OpenStack service. (string value)
+#endpoint_type = publicURL
+
+[oslo_concurrency]
+
+# Enables or disables inter-process locks. (boolean value)
+# Deprecated group/name - [DEFAULT]/disable_process_locking
+#disable_process_locking = false
+
+# Directory to use for lock files.  For security, the specified
+# directory should only be writable by the user running the processes
+# that need locking. Defaults to environment variable OSLO_LOCK_PATH.
+# If external locks are used, a lock path must be set. (string value)
+# Deprecated group/name - [DEFAULT]/lock_path
+#lock_path = <None>
```

Comment:
This may need to be defined. We don't set OSLO_LOCK_PATH, so IIRC not having one set, but not disabling it either will result in the service not running.